htdocs/class/xoopstopic.php

```diff
@@ -396,7 +400,7 @@ public function topicExists($pid, $title)
     {
         $sql = 'SELECT COUNT(*) from ' . $this->table . ' WHERE topic_pid = ' . (int)$pid . " AND topic_title = '" . trim($title) . "'";
         $rs  = $this->db->query($sql);
-        list($count) = $this->db->fetchRow($rs);
+        [$count] = $this->db->fetchRow($rs);
```

Comment:
[] in list unpacking is only allowed since PHP 7.1 

Please revert this change